spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Sequential.scala

```diff
@@ -25,10 +25,21 @@ import scala.collection.mutable.ArrayBuffer
 
 class Sequential extends MklDnnContainer with MklInt8Convertible {
 
-  val fuseConvBn = System.getProperty("bigdl.mkldnn.fusion.convbn", "false").toBoolean
-  val fuseBnRelu = System.getProperty("bigdl.mkldnn.fusion.bnrelu", "false").toBoolean
-  val fuseConvRelu = System.getProperty("bigdl.mkldnn.fusion.convrelu", "false").toBoolean
-  val fuseConvSum = System.getProperty("bigdl.mkldnn.fusion.convsum", "false").toBoolean
+  def fuseConvBn: Boolean = {
```

Comment:
I don't think we need to control at layer granularity for fusion,

Reply:
I have add the option same as graph fusion.